modules/apps/27-interchain-accounts/types/packet_test.go

```diff
@@ -132,21 +136,71 @@ func (suite *TypesTestSuite) TestGetCallbackAddresses() {
 				Data: []byte("data"),
 				Memo: "{\"callbacks\": {\"src_callback_address\": 10}}",
 			},
-			"",
+			false,
 		},
 		{
-			"memo has callbacks in json struct but does not have string value for src_callback_address key",
+			"memo has callbacks in json struct and properly formatted src_callback_address",
+			types.InterchainAccountPacketData{
+				Type: types.EXECUTE_TX,
+				Data: []byte("data"),
+				Memo: fmt.Sprintf("{\"callbacks\": {\"src_callback_address\": \"%s\"}}", expSrcCbAddr),
+			},
+			true,
+		},
+	}
+
+	for _, tc := range testCases {
+		tc := tc
+		suite.Run(tc.name, func() {
+			srcCbAddr := tc.packetData.GetSourceCallbackAddress()
+
+			if tc.expPass {
+				suite.Require().Equal(expSrcCbAddr, srcCbAddr)
+			} else {
+				suite.Require().Equal("", srcCbAddr)
+			}
+		})
+	}
+}
+
+func (suite *TypesTestSuite) TestGetDestCallbackAddress() {
```

Comment:
Seems like an odd test to have for a no-op function

Reply:
If unresolved merge conflicts cause this to return a non empty string, I'd prefer the tests to catch the issue. It's quite easy for [merge conflicts to slip through](https://github.com/cosmos/ibc-go/pull/3097#discussion_r1146205788)